message_structure.md

```diff
@@ -15,18 +15,23 @@ The following example shows common elements of a DIDComm Message. Further detail
     "created_time": 1516269022,
     "expires_time": 1516385931,
     "body": {
-    	"messagespecificattribute": "and it's value"
+    	"messagespecificattribute": "and its value"
 	}
 }
 ```
 
 ### Message Headers
 
-The attributes of a DIDComm Message at the level of its outer packaging (effectively, the "headers" of the message) are as follows:
+The attributes of a DIDComm plaintext message at the level of its outermost object (effectively, its "headers") are as follows:
 
 - **id** - REQUIRED. Message ID. The `id` attribute value MUST be unique to the sender.
 - **type** - REQUIRED. Message Type. The `type` attribute value MUST be a valid [Message Type URI](protocols.md#message-type-uri) , that when resolved gives human readable information about the message. The attributes value also informs the content of the message, for example the presence of other attributes and how they should be processed.
-- **to** - OPTIONAL. Recipient(s) identifier. The `to` attribute MUST be an array of strings where each element is a valid [DID](https://w3c.github.io/did-core/#generic-did-syntax) which identifies the recipients of the message.
+- **to** - OPTIONAL. Identifier(s) for recipients. MUST be an array of strings where each element is a valid [DID](https://w3c.github.io/did-core/#generic-did-syntax) that identifies a member of the message's intended audience.
+
+  The relationship between this array and routing is worthy of comment. A message from Alice to Bob is seen as plaintext by Alice and by Bob, and as ciphertext by all intermediaries. Thus, the value of the `to` array as seen by Bob cannot be used for routing. Instead, when Bob sees it, it offers semantics like the `To:`, `CC:`, and `BCC:` headers in email, allowing a recipient to enumerate all parties that the sender claims might see the message in plaintext. This has semantic value in application-level protocols built atop DIDComm.
+  
+  However, when the innermost message from Alice to Bob is attached as encrypted payload to a `forward` message in DIDComm's routing protocol, a mediator sees the `to` header of the `forward` message in plaintext. Thus, the `to` header in one or more containing envelopes does play a role in routing.
```

Comment:
This is incorrect, and needs to be removed.

Reply:
I largely removed this, but kept one sentence explaining why using `to` with routing doesn't make sense. I think it's simplified but clear.